.travis.yml

```diff
@@ -24,19 +18,31 @@ addons:
 before_install:
   - export CXX="g++-4.8" CC="gcc-4.8"
   - nvm install 6
-  - "export DISPLAY=:99.0"
-  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
+  - 'export DISPLAY=:99.0'
+  - >-
+    /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid
+    --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0
+    1280x1024x16 -extension RANDR
   - sh -e /etc/init.d/xvfb start
 install:
- - make travissetup
+  - make travissetup
+  - pip install pygments lxml setuptools --upgrade
+  - 'git clone https://github.com/tabatkins/bikeshed.git'
+  - pip install --editable $PWD/bikeshed
+  - bikeshed update
 script:
- - make check
-
+  - bash ./.travis-deploy.sh
 env:
   global:
-    - URL="https://w3c.github.io/mediacapture-fromelement/W3CTRMANIFEST"
-    - DECISION="https://lists.w3.org/Archives/Public/public-media-capture/2015Dec/0031.html"
-    - secure: "BlcDX7mHk2///8Ye3OEM1bJywVu5G9Ts6jnebio/NrSIsuD8JmaX//QVaxDQMjLdL4GWC9lVtcH5w7/gOvP45ZtyRwlhShGzhXHQTBeiqX7Iq0AvZkwp4aFpoSg6Omdkd9zXbmpDSvA0/XmBcrpXOHdyXASi6n7sa/G5Xq6eaR0="
-
+    - 'URL="https://w3c.github.io/mediacapture-fromelement/W3CTRMANIFEST"'
+    - >-
+      DECISION="https://lists.w3.org/Archives/Public/public-media-capture/2015Dec/0031.html"
+    - secure: >-
+        BlcDX7mHk2///8Ye3OEM1bJywVu5G9Ts6jnebio/NrSIsuD8JmaX//QVaxDQMjLdL4GWC9lVtcH5w7/gOvP45ZtyRwlhShGzhXHQTBeiqX7Iq0AvZkwp4aFpoSg6Omdkd9zXbmpDSvA0/XmBcrpXOHdyXASi6n7sa/G5Xq6eaR0=
```

Comment:
I assume that you are using this rather than configuring travis directly because you don't control the travis configuration.  I find this method cumbersome.